ajuna-test/src/sidechain.rs

```diff
@@ -0,0 +1,322 @@
+// Ajuna Node
+// Copyright (C) 2022 BlogaTech AG
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::impl_block_numbers;
+use ajuna_common::{Finished, TurnBasedGame};
+use ajuna_solo_runtime::{AccountId, Balance};
+use frame_support::parameter_types;
+use frame_system::mocking::{MockBlock, MockUncheckedExtrinsic};
+use sp_core::{Decode, Encode, H256};
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, IdentityLookup},
+};
+use sp_std::prelude::*;
+use std::marker::PhantomData;
+
+frame_support::construct_runtime!(
+	pub enum SideChainRuntime where
+		Block = MockBlock<SideChainRuntime>,
+		NodeBlock = MockBlock<SideChainRuntime>,
+		UncheckedExtrinsic = MockUncheckedExtrinsic<SideChainRuntime>,
+	{
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		AjunaBoard: pallet_ajuna_board::{Pallet, Call, Storage, Event<T>},
+		Vesting: orml_vesting,
+		Balances: pallet_balances,
+	}
+);
+
+parameter_types! {
+	pub const ExistentialDeposit: Balance = 100 * NANO_AJUNS;
+	pub const ArbitraryUpperBound: u32 = 1_000_000;
+}
+
+pub const EXISTENTIAL_DEPOSIT: Balance = 100 * NANO_AJUNS;
+
+impl pallet_balances::Config for SideChainRuntime {
+	type MaxLocks = ArbitraryUpperBound;
+	type MaxReserves = ArbitraryUpperBound;
+	type ReserveIdentifier = [u8; 8];
+	type Balance = Balance;
+	type Event = Event;
+	type DustRemoval = ();
+	type ExistentialDeposit = ExistentialDeposit;
+	type AccountStore = System;
+	type WeightInfo = ();
+}
+
+parameter_types! {
+	pub const MinVestedTransfer: Balance = 100 * MICRO_AJUNS;
+	pub const MaxVestingSchedules: u32 = 100;
+}
+
+impl orml_vesting::Config for SideChainRuntime {
```

Comment:
I don't think we need vesting in the sidechain, just in the parent chain I think

Reply:
You are right, I've moved the tests to the Node.